### DISCUSSION TITLE
Official options  under video encoding for 23.97 fps, 24 fps, and 29.97

### DISCUSSION THREAD

**Discussion by kaelsonofkrypto** [nagadomi/nunif#92] [General]
Official options  under video encoding for 23.97 fps, 24 fps, and 29.97. Thanks.

**Comment by nagadomi** [comment]
I will add this option.

**Reply by kaelsonofkrypto** [reply]
thanks

**Reply by nagadomi** [reply]
I made this change on the dev branch.
It is also editable, so you can specify any value.

Note that this is the maximum FPS setting. if the source video has a lower FPS, the original FPS will be used.

**Reply by kaelsonofkrypto** [reply]
Thanks. I'm actually new to this Github . So, just to make sure, all I have to do is just go to the Dev Branch, and I just download the gui.py file and replace it in my local files on the iw3?

**Reply by nagadomi** [reply]
I have confirmed that it works on Windows, so I merged it into the master branch.
If you are using windows-package, running `update.bat` should update the code to the latest version.

Regarding how to switch dev branch, I will create either documentation or a switching command.

**Reply by kaelsonofkrypto** [reply]
thanks , appreciate everything :)
